docker-compose.yml

```
@@ -1,15 +1,28 @@
-version: "2"
+version: "3.7"
 services:
+  nginx:
+    image: nginx:latest
+    container_name: ng01
```

lets give it a more meaningful name, like full name. 
```suggestion
    container_name: nginx
```

so commands like this one: `docker exec -ti ng01 /bin/sh` can be `docker exec -ti nginx /bin/sh`

also we are not going to have such naming structure when we want to replicate in the future.
so can we please drop the numbering and more full names please :D